CHANGELOG.next.toml

```diff
@@ -36,4 +36,8 @@ references = ["aws-sdk-rust#1079"]
 meta = { "breaking" = false, "bug" = true, "tada" = false }
 author = "rcoh"
 
-
+[[smithy-rs]]
```

Comment:
Can you update this changelog entry so that it becomes `[[aws-sdk-rust]]`?

We usually make releases in two repositories when releasing new Rust SDKs. Code is first released in `smithy-rs` and then Rust SDKs code-generated by that release are subsequently released in `aws-sdk-rust`. `[[smithy-rs]]` is for changelogs in the first release and `[[aws-sdk-rust]]` is for those in the second release.

This change, in particular, is relevant to release notes in the `aws-sdk-rust` repository as you can see from an `AWS_` prefix in the env var.